help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Gainsight PX 
+description: The Gainsight PX destination allows allows segmentation information to be pushed to Gainsight PX 
+---
+
+# Gainsight PX connection {#gainsight_px-destination}
+
+
+## Overview {#overview}
+
+[Gainsight PX](https://www.gainsight.com/product-experience/) is a product experience platform that enables product teams to understand how users use their products, collect feedback, and create in-app engagements like product walkthroughs to drive user onboarding and product adoption.
+
+>[!IMPORTANT]
+>
+>The destination connector and documentation page are created and maintained by the *Gainsight PX* team. For any inquiries or update requests, please contact them directly at *`pxsupport@gainsight.com`.*
+
```

Comment:
Please add back in the Use cases section, as described in the docs template (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#use-cases) and provide at least one clear use case for readers to understand why this destination is useful and what they can use it for.